pokemongo_bot/__init__.py

```diff
@@ -603,7 +603,25 @@ def _setup_api(self):
 
         self._print_character_info()
 
-        self.api.activate_signature("encrypt.so")
+
+
+        print os.path.dirname(__file__)
+
+
+        if _platform == "linux" or _platform == "linux2":
+            file_name = 'encrypt.so'
+        elif _platform == "darwin":
```

Comment:
since darwin and linux both use the same file_name, is it necessary to have an additional check? Wouldn't it be better to default to enscrypt.so and then only change file_name if windows were detected.